app/Http/Controllers/PetController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\PetRequest;
+use App\Http\Resources\PetResource;
+use App\Models\Pet;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Resources\Json\AnonymousResourceCollection;
+
+class PetController extends Controller
+{
+    public function index(): AnonymousResourceCollection
+    {
+        $pets = Pet::all();
+
+        return PetResource::collection($pets);
+    }
+
+    public function show(int|string $id): PetResource
```

Comment:
```suggestion
    public function show(Pew $pet): PetResource
```
Since you use Route::resource in web.php, you'll get Pet $pet and there is no need to search in db for model.
Same in update and delete, but I see that you used it in delete method.